### PR TITLE
[LIB-201]: Profile model - make some proxy to Object

### PR DESCRIPTION
**Pull request by opalczynski**

### CODE REVIEW

**Comment by 23doors** [comment]
Please correct the PR title and tests first. Otherwise it looks good but as this is pretty new and important functionality, you should add some new tests.

**Comment by opalczynski** [comment]
@23doors I know, I know. Currently trying to make it work :) When fix it - I'll write some tests.

**Comment by opalczynski** [comment]
Writin tests now, hold on with check :)

**Comment by opalczynski** [comment]
@23doors @dancio :) Hey guys, I made some serious changes here. Playing with registry. Playing with handling multiple instances and of course to make profile to work like a data object. Please check it precisely - any comment will be appreciated. The test passes - so I hope that no regression was made.

**Comment by 23doors** [comment]
It looks good to me! but then again, I wasn't all that involved with python library so I would rather get feedback from others ;)
@Zhebr @dancio could you guys take a look?

File: tests/integration_test_user_profile.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from tests.integration_test import InstanceMixin, IntegrationTest
+
+
+class UserProfileTest(InstanceMixin, IntegrationTest):
+
+    @classmethod
+    def setUpClass(cls):
+        super(UserProfileTest, cls).setUpClass()
+        cls.user = cls.instance.users.create(
+            username='JozinZBazin',
```
Comment (Zhebr) [inline tests/integration_test_user_profile.py]: :)

**Comment by Zhebr** [comment]
neat, I like the DataObject mixin approach. by the way, interesting habit of putting semicolon to finish comment

**Comment by opalczynski** [comment]
@Zhebr :) I have a many discussions about this semicolon. But to me comments sometimes looks like ended in the middle of thought. So decided to put semicolon to provide a clear message - "this comment is fully functional and won't be extended - read as it is" ;) ; #hashtag #semicolon

**Pull request merged**